.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
```

Comment:
cp definitely isn't causing any problem, it's fast.  Babel is slowish, though browserify is even slower, so I don't know if changing the order there would help – plus then we'd have to specifically opt-out browserify'd files from babel, even though in many contexts we need babel.

Though... we could probably remove babel from the server if we upgrade to Node 6.2.  I believe it has all the features we need.  And Firefox is at pretty much the same place.  So the one place we will keep needing Babel for a while is for the code delivered to the browser, which all goes through browserify.  So maybe I take that back... if we turn off babel generally, then just use it for browserify, we might speed things up really well.